mcs/tools/nunit-lite/Makefile

```diff
@@ -0,0 +1,4 @@
+thisdir = tools/nunit-lite
```

Comment:
Since we aren't shipping it, and these are basically just makefiles, I'd rather not put it at the top level of mcs.